database/migrations/2018_06_04_128500_create_app_document_table.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateAppDocumentTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('APP_DOCUMENT', function(Blueprint $table)
```

Comment:
Update these variable to use the new style. PRO_UID is process_id and USR_UID is user_id, etc.